internal/providers/amazon/eks/workflow/amazon.go

```diff
@@ -32,13 +38,20 @@ import (
 
 	"github.com/banzaicloud/pipeline/src/secret"
 
+	zapadapter "logur.dev/adapter/zap"
+
 	internalAmazon "github.com/banzaicloud/pipeline/internal/providers/amazon"
 	pkgCloudformation "github.com/banzaicloud/pipeline/pkg/providers/amazon/cloudformation"
 )
 
 // ErrReasonStackFailed cadence custom error reason that denotes a stack operation that resulted a stack failure
 const ErrReasonStackFailed = "CLOUDFORMATION_STACK_FAILED"
 
+const asgWaitLoopSleep = 5 * time.Second
+const asgFulfillmentTimeout = 2 * time.Minute
+const asgFulfillmentWaitAttempts = asgFulfillmentTimeout / asgWaitLoopSleep
```

Comment:
It might be better to define this the other way around:
```go
const asgFulfillmentWaitAttempts = 24
const asgFulfillmentTimeout = asgWaitLoopSleep * asgFulfillmentWaitAttempts
```